homeassistant/components/fivem/binary_sensor.py

```diff
@@ -0,0 +1,41 @@
+"""The FiveM binary sensor platform."""
+from homeassistant.components.binary_sensor import (
+    BinarySensorDeviceClass,
+    BinarySensorEntity,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import FiveMDataUpdateCoordinator, FiveMEntity
+from .const import DOMAIN, ICON_STATUS, NAME_STATUS
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up the FiveM binary sensor platform."""
+    server = hass.data[DOMAIN][entry.entry_id]
+
+    entities = [FiveMStatusBinarySensor(server)]
+
+    async_add_entities(entities)
+
+
+class FiveMSensorEntity(FiveMEntity, BinarySensorEntity):
+    """Representation of a FiveM sensor base entity."""
+
+    def _update_value(self):
+        self._attr_is_on = self.coordinator.data[self.type_name]
```

Comment:
Why do you use the _update value everywhere? It should work like this as well, where the data update coordinator will refresh the sensor state after a new update.

```suggestion
    @property
    def is_on(self) -> bool | None:
        """Return the state of the sensor."""
        return self.coordinator.data[self.type_name]
```